src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -478,7 +478,7 @@
                                        (:field-attributes props))))
    :component-did-mount
    (fn [{:keys [props refs]}]
-     (let [{:keys [remote render-display behavior empty-message render-suggestion on-select]} props
+     (let [{:keys [remote render-display behavior empty-message render-suggestion on-select typeahead-event]} props
```

Comment:
Maybe make the typeahead-event prop have a default of "typeahead:select" (or whichever you think is going to be more common).